imaids/blocks.py

```diff
@@ -211,6 +214,11 @@ def magnetization(self, new_magnetization):
         """Set new block magnetization vector [T]."""
         # update magnetization attribute
         self._magnetization = self._check_magnetization(new_magnetization)
+        # replace material object.
+        if self._use_default_material:
```

Comment:
@lucashfrancisco , i did not include material updating because create_radia_object already uses the subblock mag vector for easy axis and the material type is already set (from __init__) when mag setter is invoked. i dont think material updating is necessary, despite what we talked last week.

Reply:
It is probably not necessary for our use. But I believe it would be a good consistency practice to keep all Radia objects up to date, including the material Radia object referenced by the block (referenced by the imaids.Material object which is referenced by the block).
Also, this way it will not happen that one prints the object, like I did above, and the mr value shown in the material section does not correspond to the actual block remanence.

Maybe this shows an issue that the "compicated" mixed way in which the magnetization is set (vector + material  arguments) may not be ideal. But i guess this is a separate issue.

Reply:
I too found it not natural how material/magnetization is set...